### PR TITLE
Add ionice

### PR DESCRIPTION
**Pull request by jooray**
Created from https://github.com/Homebrew/homebrew/pull/21801
Fixed indentation issue as discussed

### CODE REVIEW

File: ionice.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require 'formula'
```
Comment (DomT4) [inline ionice.rb]: Can remove this. No longer required.

File: ionice.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+class Ionice < Formula
+  homepage 'https://github.com/elmarb/ionice'
```
Comment (DomT4) [inline ionice.rb]: Double quotes everywhere.

File: ionice.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  url 'https://github.com/elmarb/ionice.git', :revision => '63539195ba6aa082376a53b09a47b33c25a9b204'
+  sha1 '03effb6c344a04c9fea2f995ceb4fc8fa270d310'
```
Comment (DomT4) [inline ionice.rb]: Can't ` sha ` check a git checkout.

**Comment by jooray** [comment]
Fixed all issues

File: ionice.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+class Ionice < Formula
+  homepage "https://github.com/elmarb/ionice"
+  url "https://github.com/elmarb/ionice.git", :revision => "63539195ba6aa082376a53b09a47b33c25a9b204"
```
Comment (DomT4) [inline ionice.rb]: We don't revision scope `head-only` formula - Should just be `head "https://github.com/elmarb/ionice.git"`.

File: ionice.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  version "20120202"
```
Comment (DomT4) [inline ionice.rb]: Likewise, kill the `version`.

File: ionice.rb [hunk 6]
```diff
@@ -0,0 +1,14 @@
+class Ionice < Formula
+  homepage "https://github.com/elmarb/ionice"
```
Comment (DomT4) [inline ionice.rb]: Please add a short `desc` as well above the `homepage` - Just a simple, short line about what the tool does. Thanks!

File: ionice.rb [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  desc "start processes with lower IO priority on Mac OS X"
```
Comment (DomT4) [inline ionice.rb]: `desc` should be above the `homepage` rather than below, Thanks!

File: ionice.rb [hunk 8]
```diff
@@ -0,0 +1,14 @@
+class Ionice < Formula
+  desc "start processes with lower IO priority on Mac OS X"
+  homepage "https://github.com/elmarb/ionice"
+  url "https://github.com/elmarb/ionice.git"
```
Comment (DomT4) [inline ionice.rb]: This needs to be `head` rather than `url`.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by bfontaine** [comment]
We’re deprecating this repo because head-only formulae are a bad experience for our users: they frequently break and we don’t have any automatic update mechanism for them. You can however maintain the formula in your own tap: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md

**Comment by ibalashov** [comment]
Is it available in any tap?

**Pull request closed without merging**